jvm-libs/linea/metrics/micrometer/src/main/kotlin/net/consensys/linea/metrics/micrometer/DynamicTagTimerCapture.kt

```diff
@@ -15,7 +14,7 @@ import java.util.function.Function
  * nanosecond-level measurements. Related issue:
  * https://github.com/micrometer-metrics/micrometer/issues/535
  */
-class DynamicTagTimerCapture<T> : AbstractTimerCapture<T>, TimerCapture<T> {
+class DynamicTagTimerCapture<T> : AbstractTimerCapture<T> {
```

Comment:
You can improve this class by onboarding [my change ](https://github.com/Consensys/linea-monorepo/pull/1107/files#diff-4b230feff290192bebef11dee694282e81d195b1e9abbb3dafab9cb953638a9dR11)to split it in to a Builder and the `TimerCapture`. But probably this class needs a redesign anyway. Up to you